src/components/Account.jsx

```diff
@@ -19,9 +32,17 @@ export const Account = ({ currentUser }) => {
       </div>
       <div id="order-products">
         <h3>My Orders</h3>
-        {
-          // MAP ORDERS + ORDER PRODUCTS HERE
-        }
+        <ul>
+          {orderList && orderList.length > 0
```

Comment:
remove first orderList and move the && after the boolean check, and then remove null

Reply:
orderList.length > 0 && ...map()...